### ISSUE TITLE
Question regarding weight hyperparameter search

### ISSUE THREAD

**Issue by SoongE** [YBZh/DMN#2]
Thank you for your insightful paper and for sharing your code.

I have a question regarding the selection of the weight hyperparameters, (\alpha_1, \alpha_2, \alpha_3) in Equation (12) of your [paper](https://openaccess.thecvf.com/content/CVPR2024/papers/Zhang_Dual_Memory_Networks_A_Versatile_Adaptation_Approach_for_Vision-Language_Models_CVPR_2024_paper.pdf). 
Could you clarify whether these hyperparameters were determined through the search on the test set or validation set?
Are all the results reported in the paper based on hyperparameter searches conducted using the test set?
https://github.com/YBZh/DMN/blob/c6c218c70ebbc701b31cc3d7f7b9597117daad8e/dmn_main.py#L829

I appreciate your help and look forward to your response.
Thank you!

**Comment by YBZh** [comment]
Thanks for your interest. The best results reported in the paper are achieved by searching hyperparameters on the test set. 

To facilitate its practical application, we also report results with fixed \alpha_1, \alpha_2, \alpha_3 for the reference, which still outperforms the competitors.

**Comment by SoongE** [comment]
Thank you for your quick response to my question.

If you don’t mind, could we obtain the few-shot performance score of the training-free DMN on the 11 datasets for graphing in Figure A9?

**Comment by YBZh** [comment]
Figure A9 does not contain results under the training-free setting. Could you clarify your requirements?

**Comment by SoongE** [comment]
You are right, that was my mistake.

I need the few-shot classification results of DMN-TrainFree on ResNet50, as shown in Figure A7.

**Comment by SoongE** [comment]
If you don't mind, I would be very happy to have the results of both A7 and A9. :)

**Comment by YBZh** [comment]
[training_free_res50.xlsx](https://github.com/user-attachments/files/17584553/training_free_res50.xlsx)
[training_required_res50.xlsx](https://github.com/user-attachments/files/17584562/training_required_res50.xlsx)
[training_required_vit.xlsx](https://github.com/user-attachments/files/17584563/training_required_vit.xlsx)
[python.zip](https://github.com/user-attachments/files/17584579/python.zip)

The results in xlsx are attached, also with the code for figure plot. I hope this can help you.

**Comment by SoongE** [comment]
Thank you so much, it will be helpful in my research.

I'll close the issue.
